src/cli.js

```diff
@@ -32,7 +34,7 @@ let cli = {
       // Show help
       console.log(options.generateHelp());
     } else {
-      nibbler.setExtensions(extensions);
+      nibbler.setExtensions(extensions, config);
```

Comment:
I think we need to also consider what happens if a custom config is not specified.  We can rely on there always being an `extensions`, because it has a default.  But I think that `config` will be `undefined` if the user does not specify it (that should be verified).

Perhaps it would be best to rename `setExtensions` to something like `configure`, and pass it an object, something like:

```js
let configuration = {extensions: extensions};
if (config) configuration.configFile = config;  // wow that's a lot of "config" ;)
nibbler.configure(configuration);
```

What would you think of doing something like that?